src/views/Playgrounds/PlaygroundForm.tsx

```diff
@@ -0,0 +1,120 @@
+import React, { useState } from 'react'
+import { useForm, Controller } from 'react-hook-form'
+import {
+  Button as _Button,
+  Text,
+  FormField,
+  TextField,
+  Select,
+  Checkbox,
+  Textarea,
+  HeaderTextField,
+} from '@/shared/components'
+import { textFieldValidation, requiredValidation } from './formValidationOptions'
+import { SelectedItem } from '@/shared/components/Select'
+import styled from '@emotion/styled'
+
+const items: SelectedItem[] = [
+  { name: 'Public (Anyone can see this video', value: 'public' },
+  { name: 'Private', value: 'private' },
+]
+
+type Inputs = {
+  title: string
+  videoSelect: string
+  header: string
+  check: string
+  textarea: string
+}
+
+const PlaygroundForm = () => {
+  const { register, handleSubmit, control, setValue, errors } = useForm<Inputs>({
+    shouldFocusError: false,
+  })
+  const [selectedItem, setSelectedItem] = useState<SelectedItem | null>(null)
+  const [checkboxValue, setCheckboxValue] = useState(false)
```

Comment:
Can we get rid of these two guys? 
Although I tried myself to do this and failed with that. But still think that we could remove at least the checkbox handler - maybe we should do some changes in Checkbox itself

Reply:
:+1: Most certainly, we shouldn't need to keep any state on our own. Take a look here: https://react-hook-form.com/get-started/#IntegratingControlledInputs

Or is there something stopping us from doing that?